src/core/components/dataList/dataListContainer/dataListContainer.tsx

```diff
@@ -49,21 +63,24 @@ export const DataListContainer: React.FC<IDataListContainerProps> = (props) => {
     const displayItems =
         state === 'fetchingNextPage' || state === 'idle' || state === 'loading' || state === 'filtered';
 
+    const isError = state === 'error';
     const isEmpty = state === 'idle' && childrenItemCount === 0;
     const isEmptyFiltered = state === 'filtered' && childrenItemCount === 0;
 
+    const applyLayoutClassName = !isError && !isEmpty && !isEmptyFiltered;
+
     useEffect(() => {
         setChildrenItemCount(childrenItemCount);
     }, [setChildrenItemCount, childrenItemCount]);
 
     return (
         <div
-            className={classNames('flex flex-col gap-2', className)}
+            className={classNames('flex flex-col gap-2', applyLayoutClassName && layoutClassName, className)}
```

Comment:
This does mean though that you don't actually have a way to always add a classname to the container (which is fine, I think).

Reply:
Users can also use the normal `className` property to set the classnames of the data-list-container, these will always be set